### PR TITLE
Added xero.gg support.

### PR DESCRIPTION
**Pull request by Dekirai**
Showcase: [Showcase](https://streamable.com/7c39tj)
Website: [xero.gg](https://xero.gg/)
What it is: xero.gg is a player and clan database for the private server called "S4 Xero" of the base game "S4 League".
What the presence does: The presence mainly shows what profile or clan you are currently looking at or what news you are currently reading. More features included and might follow.

Screenshots of the presence: 
image
image
image

### CODE REVIEW

**Review by ririxichan — CHANGES_REQUESTED** [review]
upload proof of working presence

**Review by ririxichan — APPROVED** [review]

**Comment by Dekirai** [comment]
Uhh how do I fix this Codacy stuff?

**Comment by Dekirai** [comment]
Whats wrong with the codacy?
How do I fix this?

**Comment by Dekirai** [comment]
I don't understand why I'm getting the codacy errors now.
It worked before and I haven't touched anything where it throws the error.
How would I fix this? I don't understand it.

**Review by ririxichan — CHANGES_REQUESTED** [review]
You don't need let variables for them, because you only use these variables in if statements so please change it to const.

**Comment by Dekirai** [comment]
I hope it's okay now..

File: websites/X/Xero.gg/presence.ts [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+      document.querySelector(
+        "#player-profile-header-heading > div.medium.normal-color-name"
+      ).textContent,
+      document.querySelector(
+        "#clan-data-container > div > div.col-sm-3 > div > div.xero-pane-body > ul > li"
+      ).lastChild.textContent,
+      document.querySelector(
+        "#clan-data-container > div > div.col-sm-9 > div:nth-child(2) > div > div > a > div.media-body.ml-2 > div.bold"
+      ).textContent
+    ];
```
Comment (Bas950) [inline websites/X/Xero.gg/presence.ts:63]: thats an array, thats now how you merge consts

```
const var1 = obj,
var2 = obj2;
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by Dekirai** [comment]
Okay I hope I finally understood it now.
I feel so stupid, I'm so sorry.

**Review by ririxichan — APPROVED** [review]

**Comment by Bas950** [comment]
just to be sure please upload new proof of it working

**Comment by Dekirai** [comment]
Ah sure, 1 sec

**Comment by Dekirai** [comment]
@Bas950 I replaced the Showcase Link with a new video (You can also see the current date and time of windows)

**Review by Bas950 — APPROVED** [review]

**Pull request merged**